NorthstarDLL/plugins/plugins.cpp

```diff
@@ -129,7 +129,7 @@ std::optional<Plugin> PluginManager::LoadPlugin(fs::path path, PluginInitFuncs*
 	if (strcmp(manifestJSON["api_version"].GetString(), std::to_string(ABI_VERSION).c_str()))
 	{
 		NS::log::PLUGINSYS->error(
-			"'{}' has an incompatible API version number '{}' in its manifest. Current ABI version is '{}'", pathstring, ABI_VERSION);
+			"'{}' has an incompatible API version number in its manifest. Current ABI version is '{}'", pathstring, ABI_VERSION);
```

Comment:
Wait, I'm confused.

Looking at old version, it looks like `ABI_VERSION` held version number of plugin that was read when in reality it's defined in `NorthstarDLL/plugins/plugin_abi.h` as the plugin version of the client and NOT the current plugin.

I guess old log string was just plain wrong then, lol

Reply:
There's multiple log calls in plugins that are just plain wrong

Reply:
I didn't add new log calls

Reply:
should this get resolved or not?

Reply:
I moved the change to #591 which I'll merge first and then resolve the conversation here ^^